### PR TITLE
actions: add several actions

### PR DESCRIPTION
**Pull request by gibiw**
actions: add several actions
--
- run-create
- run-complete
- report
- env-create
- milestone-create

### CODE REVIEW

File: run-create/README.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+**Required**. Title of the test run.
+
+### description
+
+Description of the test run.
+
+### environment
+
+Environment of the test run.
```
Comment (NickVolynkin) [inline run-create/README.md:25]:
```suggestion
Environment of the test run (slug). For example, 'dev', 'feature-123', 'production'.
```

File: run-create/README.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+### milestone
+
+Milestone of the test run.
```
Comment (NickVolynkin) [inline run-create/README.md:29]:
```suggestion
Milestone of the test run, specified by ID.
```

File: run-create/README.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+  - uses: qase-tms/gh-actions/run-create
+    id: create
+    with:
+      token: ${{ secrets.QASE_TOKEN }}
+      project: PRJ_CODE
+      title: Test Run Title
+      description: Test Run Description
+      environment: Production
+      milestone: Release 1.0
```
Comment (NickVolynkin) [inline run-create/README.md:59]: ORLY, can we specify milestones by their title?

File: run-create/README.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+      plan: Regress
```
Comment (NickVolynkin) [inline run-create/README.md:60]: Same with plans, I don't see it in the API. Unless it's done by the CLI app.

https://developers.qase.io/reference/create-run

File: run-create/action.yaml [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+    required: false
+  environment:
+    description: "Environment of the test run"
+    required: false
+  milestone:
+    description: "Milestone of the test run"
+    required: false
+  plan:
+    description: "Plan of the test run"
+    required: false
```
Comment (NickVolynkin) [inline run-create/action.yaml:25]: Inputs can have types declared: sting, int, bool. If the type is wrong, GH will show an error when a workflow is pushed, which is better and earlier than if CLI throws an error trying to get an environment by ID.

File: run-create/action.yaml [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+  verbose:
+    description: "Enable verbose output"
+    required: false
+    default: "false"
```
Comment (NickVolynkin) [inline run-create/action.yaml:29]: can be bool with literal `false` value

File: run-create/README.md (hunk 2 above)
Comment (gibiw) [reply run-create/README.md:29]: Do you want to specify ID of env, milestone, plan?

File: run-create/action.yaml (hunk 5 above)
Comment (gibiw) [reply run-create/action.yaml:25]: I don't see any information about types: https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputs

File: run-create/README.md (hunk 1 above)
Comment (gibiw) [reply run-create/README.md]: Fixed

File: run-create/README.md (hunk 2 above)
Comment (gibiw) [reply run-create/README.md]: Fixed

File: run-create/README.md (hunk 4 above)
Comment (gibiw) [reply run-create/README.md]: Fixed

File: run-create/action.yaml (hunk 5 above)
Comment (gibiw) [reply run-create/action.yaml]: Fixed

File: run-create/action.yaml (hunk 6 above)
Comment (gibiw) [reply run-create/action.yaml:31]: Fixed

**Review by NickVolynkin — APPROVED** [review]

**Review by NickVolynkin — APPROVED** [review]

**Pull request merged**